src/metadata.ts

```diff
@@ -251,9 +259,9 @@ export class MetadataModule {
                     getFolderEntries(projectId, 'metric'),
                     this.getDimensions(projectId)
                 ])
-                .then(([facts, metrics, attributes]) => {
-                    return { fact: facts, metric: metrics, attribute: attributes };
-                });
+                    .then(([facts, metrics, attributes]) => {
+                        return { fact: facts, metric: metrics, attribute: attributes };
```

Comment:
Could you please simplify this to `{ fact, metric, attribute }`?

Reply:
Good catch, updated. Thanks!